### ISSUE TITLE
Not correct nested result

### ISSUE THREAD

**Issue by Ewg777** [JiriTrecak/Laurine#25]
Not issue but add some warning would be good. 
Localizable.strings

```
"profile.phone" = "Some title"
"profile.phone.number" = "Your phone number!"
```

Laurine will create without any warnings

```
                struct phone {

                                    /// Base translation: Your phone number!
                    static var number: String = "profile.phone.number".localized

                }
```

**Comment by JiriTrecak** [comment]
Hey!

Thank for the feedback, and you are right - it is actually already on my roadmap. While it can provide warning, usually it is not that good because this happens quite often. So instead, the laurine in next version will generate ->

```
"profile.phone" = "Some title"
Profile.Phone.StringValue
"profile.phone.number" = "Your phone number!"
Profile.Phone.Number
```

And the StringValue will become reserved keyword for last part of the struct chain.

Thank you and have a nice day!
Sincerely
JT

**Comment by JiriTrecak** [comment]
This will be fixed in upcoming Laurine 2.0. Please refer to #52 for any further comments.
